_acs-aem-commons/features/fast-action-manager/index.md

```diff
@@ -5,27 +5,106 @@ description: aka FACEMELTER
 date: 2016-04-25
 redirect_from: /acs-aem-commons/features/fast-action-manager.html
 feature-tags: backend-dev content-migration
-tags: acs-aem-commons-features
-initial-release: 2.4.0/3.0.0
+tags: updated
+initial-release: 2.12.0/3.9.0
```

Comment:
I could be wrong, but I'm pretty sure this only applies to 3.9.0, not 2.12.0, especially since 2.12.0 doesn't require Java 8 or 6.2.